### PR TITLE
Devops 1164

### PR DESCRIPTION
**Pull request by pickgr**
Finally got around to testing this end to end with manolo-stg.  I believe it's working how we want it to now.

I'll create a new release to include it.

### CODE REVIEW

**Comment by cristobal23** [comment]
I had a similar changes staged, so I just pushed them to your branch.  All that is left is updating the CHANGELOG, which I am working on right now.

**Comment by cristobal23** [comment]
Since this `suspended_process` breaks compatibility with terraform versions < 0.8.0 this will be a major version release of this module.

**Comment by pickgr** [comment]
Sounds good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,14 @@
 * Consider coding `ebs_optimized` against list of [ebs-optimized instances](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSOptimized.html).
 * Expose ephemeral block device support
 
+## 0.3.0 (Jan 24, 2016)
+
+#### BACKWARDS INCOMPATIBILITIES / NOTES:
+* Exposed `suspended_processes` to disable Auto Scaling Group processes.  This resource is only available in versions of terraform >= 0.8.0.
+
+#### IMPROVEMENTS:
+* Added EC2 instance tag `env` to descrive the environment of the stack.
```
Comment (pickgr) [inline CHANGELOG.md]: typo here - descrive -> describe

**Pull request merged**